### PR TITLE
Coding practice 18 is added

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/SubstringBreaker.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package codingPractice;
+import java.util.HashSet;
```
Comment (prashantkawa) [inline simple/src/codingPractice/SubstringBreaker.java:2]: add space between package and import statement

File: simple/src/codingPractice/SubstringBreaker.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import java.util.Set;
+
+public class SubstringBreaker {
+
+
```
Comment (prashantkawa) [inline simple/src/codingPractice/SubstringBreaker.java:7]: remove extra space

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: simple/src/codingPractice/SubstringBreaker.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/SubstringBreaker.java:3]: Done

File: simple/src/codingPractice/SubstringBreaker.java (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/SubstringBreaker.java]: Done

**Review by anushreeSonak — APPROVED** [review]

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**